.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/30-seconds-of-php-code.iml" filepath="$PROJECT_DIR$/.idea/30-seconds-of-php-code.iml" />
+    </modules>
+  </component>
+</project>
```

Comment:
This file is not necessary.
You have to follow the CONTRIBUTING guide to do the contribution.